hassio/src/hassio-panel-router.ts

```diff
@@ -23,6 +24,7 @@ class HassioPanelRouter extends HassRouterPage {
   @property({ type: Boolean }) public narrow!: boolean;
 
   protected routerOptions: RouterOptions = {
+    initialLoad: () => this._redirects(),
```

Comment:
Use the `beforeRender` option instead. 

```suggestion
    beforeRender: (page: string) => page === "snapshots" ? "backups" : undefined,
```

Reply:
I thought the `routeOption` could just be a string to redirect?